package.json

```diff
@@ -44,6 +44,7 @@
     "classnames": "2.3.1",
     "clone-deep": "4.0.1",
     "date-fns": "2.29.1",
+    "fast-deep-equal": "3.1.3",
```

Comment:
I only picked this lib because it was already in our yarn.lock but happy to switch to something else if there's anything better.